api/v1beta1/common.go

```diff
@@ -19,6 +19,7 @@ package v1beta1
 type extraConfigmapsMounts struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength:=253
```

Comment:
Is this something we want to add to other name parameters as well (e.g., `stepName`). Why 253? I'm just curious.

Reply:
no clue why 253, i copied it from the global operator .. i don't know if we want to add it to other name parameters, do we? I didn't notice other operators doing this